src/sentry/migrations/0482_denormalize_user_email.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.28 on 2023-05-30 17:29
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0481_add_pull_request_comment_table"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    ALTER TABLE "sentry_organizationmember" ADD COLUMN "user_email" VARCHAR(75) NULL;
+                    """,
+                    reverse_sql="""
+                    ALTER TABLE "sentry_organizationmember" DROP COLUMN "user_email";
+                    """,
+                    hints={"tables": ["sentry_organizationmember"]},
```

Comment:
Should we split this migration into its own PR to ensure that we don't have another potential migration revert scenario?

Reply:
I feel highly confident this change will be safe, and the reversal of this migration will be safe.

Reply:
We never run reverse migrations, nor do we have tooling to do so. Column add operations are generally safe though.